torch/onnx/_internal/jit_utils.py

```diff
@@ -321,3 +321,13 @@ def get_device_from_value(value: _C.Value) -> Optional[torch.device]:
         return None
     tensor_type = typing.cast(_C.TensorType, value.type())
     return tensor_type.device()
+
+
+@_beartype.beartype
+def _parse_node_kind(kind: str) -> Tuple[str, str]:
```

Comment:
Make it public and include a brief docstring?

Reply:
jit_utils.py is internal already so no need to hide the function when it is consumed by other modules. All `_` functions should be use only in their own modules (which we are not doing a good job in symbolic helper)

Reply:
Got it now!

Reply:
Done